docs/pages/api.rst

```diff
@@ -448,17 +450,24 @@ Signature: ``Collection::compact(...$values);``
 contains
 ~~~~~~~~
 
-Check if the collection contains one or more values.
+Check if the collection contains one or more values. Note that if multiple values are passed the operation
```

Comment:
Maybe we could copy paste the warning box that explain this? For example, see the DropWhile operation: https://loophp-collection.readthedocs.io/en/latest/pages/api.html#dropwhile

Reply:
I thought I remembered a similar one! Sure can!

Is this valid across all operations btw? Variadic parameters acting as `OR`

Reply:
Thanks !

And yeah, I'm trying to make all the variadic as `OR` indeed.

Reply:
added now. Had to make a tweak to specify that in this case you cannot use it as a logical AND by calling in succession because the collection will have just a `bool` after the first call